dubbo-common/src/main/java/org/apache/dubbo/config/context/AbstractConfigManager.java

```diff
@@ -551,7 +551,18 @@ public <T extends AbstractConfig> List<T> loadConfigsOfTypeFromProps(Class<T> cl
         if (this.getConfigs(cls).isEmpty()) {
             // load single config
             List<Map<String, String>> configurationMaps = environment.getConfigurationMaps();
-            if (ConfigurationUtils.hasSubProperties(configurationMaps, AbstractConfig.getTypePrefix(cls))) {
+
+            // check type prefix.
+            String typePrefix = AbstractConfig.getTypePrefix(cls);
+            boolean hasSubProperties = ConfigurationUtils.hasSubProperties(configurationMaps, typePrefix);
+
+            // check ${prefix}.enabled property configuration.
+            boolean metricsEnabled = configurationMaps.stream().anyMatch(configurationMap -> {
+                String key = typePrefix + ".enabled";
```

Comment:
Should not hard code here

Reply:
I updated the code, extracted it into the commonconstant definition. pls review again. @AlbumenJ